master/buildbot/www/config.py

```diff
@@ -58,6 +58,8 @@ def renderIndex(self, request):
         config['title'] = self.master.config.title
         config['titleURL'] = self.master.config.titleURL
         config['multiMaster'] = self.master.config.multiMaster
+        if "wamp" in self.master.config.protocols:
+            config['wamp'] = self.master.config.protocols['wamp']
```

Comment:
True, but only as a default, and because it's a historical precedent.  I don't really like it.  Ideally this would be configured more like:

```
c['wamp'] = {
  'url': '...',
}

c['protocols'] = {
    'wamp': {},
    'pb': {'port': 9999},  # for slaves we haven't upgraded yet
}
```

I think this would be a bit clearer if the implementation is split into a few PRs as I suggested -- the first "add WAMP support" commit wouldn't touch `c['protocols']`, saving that for the slave PR.